mne/decoding/time_gen.py

```diff
@@ -914,3 +953,358 @@ def time_generalization(epochs_list, clf=None, cv=5, scoring="roc_auc",
                       for train, test in cv)
     scores = np.mean(scores, axis=0)
     return scores
+
+
+class TimeDecoding(_Decoder):
+    """
+    Cross-validate a series of estimators across every time point
+    to decode the event_id of each epoch.
```

Comment:
it's doing less hence is simpler / faster but tries to reuse as much code
as possible the GAT utilities.